formulaire/src/main/java/fr/openent/formulaire/service/impl/FormulaireApplicationStorage.java

```diff
@@ -0,0 +1,115 @@
+package fr.openent.formulaire.service.impl;
+
+import fr.openent.form.core.models.ShareMember;
+import fr.openent.form.helpers.FutureHelper;
+import fr.openent.formulaire.service.FormService;
+import fr.wseduc.webutils.DefaultAsyncResult;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.core.Promise;
+import io.vertx.core.eventbus.EventBus;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import org.entcore.common.notification.TimelineHelper;
+import org.entcore.common.sql.Sql;
+import org.entcore.common.sql.SqlResult;
+import org.entcore.common.storage.FileInfos;
+import org.entcore.common.storage.StorageException;
+import org.entcore.common.storage.impl.PostgresqlApplicationStorage;
+import org.entcore.common.user.UserInfos;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static fr.openent.form.core.constants.Constants.FORMULAIRE;
+import static fr.openent.form.core.constants.Fields.*;
+import static fr.openent.form.core.constants.Fields.BODY;
+import static fr.openent.form.core.constants.Tables.*;
+
+public class FormulaireApplicationStorage extends PostgresqlApplicationStorage {
+    private final static Logger log = LoggerFactory.getLogger(FormulaireApplicationStorage.class);
```

Comment:
```suggestion
    private static final Logger log = LoggerFactory.getLogger(FormulaireApplicationStorage.class);
```
spec' java sur les Modifiers (les classes, attributs etc..)